demos/projects/ESPRESSIF/adu/main/azure_iot_freertos_esp32_pnp_model.c

```diff
@@ -254,6 +256,17 @@ void vHandleWritableProperties( AzureIoTHubClientPropertiesResponse_t * pxMessag
                 return;
             }
 
+            LogInfo( ( "Verifying JWS Manifest" ) );
+            uint32_t ulJWSVerify = JWS_ManifestAuthenticate( (char*)xAzureIoTAduUpdateRequest.pucUpdateManifest,
```

Comment:
Since the parameter to function is const, shouldn't the typecast be (const char*).  Though please check out my feedback about this in `JWS_ManifestAuthenticate` header before changing as I'm not sure char* is right.